### ISSUE TITLE
Spurious stacktrace while operating the daemon

### ISSUE THREAD

**Issue by ribasushi** [ipfs/go-ipfs#3286]
I was playing with the daemon built today from https://github.com/ipfs/go-ipfs/commit/67a1b3e1b and somewhere 3 minutes after it started I got the [following stacktrace](https://ipfs.io/ipfs/Qmb6UVQ8iKvD51WkhDjn6s1B2Z9RLE5TW3DbqFaEsFwtrv) on STDERR. The daemon kept functioning afterwards, in fact I used the same process instance to publish the stacktrace itself.

I was encouraged to open an issue with this:

```
<lgierth> interesting stacktrace
<lgierth> mind filing an issue with that? ^
```

**Comment by whyrusleeping** [comment]
This has been resolved in master, at least since: 53d47669d, but possibly on an earlier update of go-cid as well.

**Comment by whyrusleeping** [comment]
Thanks for reporting @ribasushi !
